README.md

```diff
@@ -59,60 +72,55 @@ You'll need this package structure to construct a provider package repo:
 ```
 
 
-## Development Standards
+## Part 2: Development Standards
 
-### Building Provider Package
+If you followed the formatting guidelines above, you're now ready to start editing files to include standard package functionality.
 
-Most of what you need is included in `setup.py` and ready to customize. You may use `setup.cfg` as a valid alternative.
+### Python Packaging Scripts
 
-### Provider Readmes
+Your `setup.py` file should contain all of the appropriate metadata and dependencies required to build your package. Use the [sample `setup.py` file](https://github.com/astronomer/airflow-provider-sample/blob/main/setup.py) in this repository as a starting point for your own project.
 
-Readmes should contain top-level documentation about the provider's service, how to build a connection to the service from Airflow, what modules exist within the package, what dependency versions the provider has been tested with, and how the repository maintainers would like folks to contribute.
+If some of the options for building your package are variables or user-defined, you can specify a `setup.cfg` file instead.
 
-#### Managing Dependencies
+### Managing Dependencies
 
-When building providers, these guidelines will help you avoid potential for dependency conflicts.
+When building providers, these guidelines will help you avoid potential for dependency conflicts:
```

Comment:
This is a point where it'd be helpful to call these out as requirements or best practices.